napari/layers/points/_tests/test_points.py

```diff
@@ -598,6 +598,35 @@ def test_edge_color_cycle():
     )
 
 
+def test_adding_value_edge_color_cycle():
+    """ Test that adding values to properties used to set an edge color cycle
+    and then calling Points.refresh_colors() performs the update and adds the
+    new value to the edge_color_cycle_map.
+
+    See: https://github.com/napari/napari/pull/1034
```

Comment:
```suggestion
    See: https://github.com/napari/napari/issues/988
```

Reply:
I think it's good here to point to the issue that it's testing, rather than the PR that added the code — which will be visible with `git blame` anyway.